kolibri/plugins/device/assets/src/views/DeviceAppBarPage.vue

```diff
@@ -0,0 +1,37 @@
+<template>
+
+  <AppBarPage :title="title">
+
+    <template #subNav>
+      <DeviceTopNav />
+    </template>
+
+    <DeprecationWarningBanner />
+
+    <div style="margin-top: 1em">
```

Comment:
Should we always include this margin, or maybe add it as a margin bottom in the deprecation warning banner instead, and put it on an element that only appears if the warning appears?